test/e2e/events.e2e.js

```diff
@@ -51,4 +51,20 @@ describe('ContactHub', () => {
     });
   });
 
+  describe('getEvents', () => {
+    it('retrieves all Events for a Customer using filters', async () => {
+      const dateFrom = new Date('2017-02-28');
+      const filters = {
+        mode: 'ACTIVE',
+        context: 'WEB',
+        dateFrom
+      };
+      const events = await ch.getEvents(cid, filters);
+      const areWebEvents = events.reduce((acc, v) => acc && v.context === 'WEB', true);
+      expect(areWebEvents).toBe(true);
```

Comment:
This expectation is likely to be always true, because all the test events in the test workspace have `type=WEB`. Ideally we should have a range of different event in the database, but as a temporary solution I would manually create an event with a different `type`, and use that type for this test.